sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/TemporalityUtils.java

```diff
@@ -15,25 +14,14 @@ private TemporalityUtils() {}
   /**
    * Resolves which aggregation temporality to use for a given measurement.
    *
-   * @param supported All aggregation temporalities supported by the exporter.
    * @param preferred The preferred temporality of the exporter.
    */
-  static AggregationTemporality resolveTemporality(
-      EnumSet<AggregationTemporality> supported, @Nullable AggregationTemporality preferred) {
-    // Next assume preferred should always win.
+  static AggregationTemporality resolveTemporality(@Nullable AggregationTemporality preferred) {
+    // If preferred temporality is provided respect it
     if (preferred != null) {
       return preferred;
```

Comment:
we could just do a simple one-liner for this whole method now, right?
`return preferred == null ? AggregationTemporality.CUMULATIVE : preferred;`

Reply:
Yes. Will update.